assets/css/components/hero.css

```diff
@@ -1,3 +1,11 @@
 .min-height-mobile {
   min-height: 398px;
 }
+
+.gray-overlay {
+  background: rgba(0, 0, 0, 0.25);
+}
+
+.with-padding {
+  padding: 3rem 1.5rem;
+}
```

Comment:
Hmm the css for this file still ends up being global, could we have a more restrictive name for it?

Reply:
actually this might be fixed when i port over to the one with helpers as i'll be able to use the stuff there (either that or i do inline cos it's very localised)

Reply:
eh nvm, wdyt about `hero-infobox-padding`